### PR TITLE
Updated InstallGuide to include newer missions and to give better war...

### PR DESCRIPTION
**Pull request by SgStapleton**
...ning in ISIS SPICE Web Service section. References #5218

### CODE REVIEW

File: isis/src/docsys/documents/InstallGuide/InstallGuide.xml [hunk 1]
```diff
@@ -386,12 +386,11 @@ you must <em>not</em> upgrade the ISIS Data Files!!!
         </p>
 
         <span style="font-size:120%; color:red; font-weight:bold">
-          WARNING: Some missions do not currently support calibration when using the SPICE Web
-          Service exclusively: these include LRO, MRO, Viking, MGS, and MER. These calibration
-          programs are lronaccal, lrowaccal, mical, moccal, ctxcal, hical, vikcal, mocevenodd, and
-          mocnoise50. Some programs designed to run an image from ingestion through the mapping
-          phase do not have an option to use the SPICE Web Service. Known programs are thmproc,
-          mocproc, hicalproc, and mdisproc.
+          WARNING: Some instruments require mission data to be present for callobration, which may
```
Comment (kberryUSGS) [inline isis/src/docsys/documents/InstallGuide/InstallGuide.xml]: calibration, rather than callobration?

Comment (kberryUSGS) [reply isis/src/docsys/documents/InstallGuide/InstallGuide.xml]: Er, also, I didn't intend to make myself "The Reviewer" for your PR, I just noticed the typo and wanted to point it out.

**Comment by kberryUSGS** [comment]
[Actually PR the QT change into dev.](https://trello.com/c/yUC4UuGu/183-actually-pr-the-qt-change-into-dev)

**Comment by SgStapleton** [comment]
Will get that fixed. Thank you Kristin!

**Comment by kberryUSGS** [comment]
@SgStapleton If you'd like, you can update an already existing PR rather than closing one and creating another. When you push your updates to `SgStapleton:m05218`, the existing PR will automatically update with the new changes and anyone who made comments will get notified that you made new changes. For example, see the back-and-forth here: https://github.com/USGS-Astrogeology/ISIS3/pull/96. (I think doing things this way is less confusing, since it keeps the review comments and associated changes together!)

**Pull request closed without merging**